ONSdigital/census-rm-toolbox
# Add qid link bulk processor

# Checklist
Reminder: If any change made to this repo affects the acceptance tests then the version pin in the Pipfile in [census-rm-acceptance-tests]() needs to be updated.

# Motivation and Context
Add new bulk processor to link a qid to a case.  This is "meant" to be a one off, so hasn't been added to the automated bulk processing script.

# What has changed
- New bulk processor to link a qid to a case
- New validator to check ensure that  Census QID can't be linked to a CCS case and vice versa.
- Tests

# How to test?
- Run the tests
- Run the corresponding terraform and k8s branches (see links below)
- Test in your env using a file in the correct format (see README)

# Links
- https://trello.com/c/LCqKnTGw
- https://github.com/ONSdigital/census-rm-kubernetes/pull/421
- https://github.com/ONSdigital/census-rm-terraform/pull/314
